tests/js/harness.js

```diff
@@ -7,10 +7,15 @@ require("gpii-express");
 require("../../");
 
 
-fluid.defaults("gpii.pouch.tests.harness", {
+fluid.defaults("gpii.test.pouch.harness", {
     gradeNames: ["fluid.component"],
-    port:       6789,
-    baseUrl:    "http://localhost:6789/",
+    databases: {
```

Comment:
See here for extent of the remaining duplication
https://github.com/amb26/kettle/blob/KETTLE-32/tests/DataSourcePouchDBTests.js#L22